source/index.d.ts

```diff
@@ -277,16 +241,101 @@ export const supportsColor: ColorInfo;
 export const chalkStderr: typeof chalk;
 export const supportsColorStderr: typeof supportsColor;
 
+export {
+	ModifierName,
```

Comment:
You can use `export` for `modifierNames` and the other instead of redefining them and duplicating all the docs.